benchmark/runner.js

```diff
@@ -24,7 +24,16 @@ if (optimist.argv.help) {
 if (optimist.argv.bundle) {
   var bundle = require("browserify")({debug: true});
   bundle.require(path.resolve(__dirname, "browser-runner.js"), {expose: "jsdom-browser-runner"});
-  bundle.bundle().pipe(fs.createWriteStream(__dirname + "/browser-bundle.js"));
+
+  bundle.bundle()
+    .pipe(fs.createWriteStream(__dirname + "/browser-bundle.js"))
+    .on("finish", function () {
+      let uri = path.resolve(__dirname, "browser-runner.html");
```

Comment:
[URL, not URI](https://url.spec.whatwg.org/#goals) (also in commit message)

We have a toFileUrl utility somewhere you can use for this?